### DISCUSSION TITLE
Successful implementation of Clover on a legacy 2009 Gigabyte - AMD system !

### DISCUSSION THREAD

**Discussion by Morbius01** [CloverHackyColor/CloverBootloader#491] [General]
After several weeks of research, I finally managed to successfully  install Clover on a 2009 (!) legacy BIOS (non UEFI) system.

My system :
* MB : Gigabyte MA790XT-UD4P (2009)
* CPU : AMD Phenom II X4 955 (2009)
* SSD : TRANSCEND TS1MTE220S 1TB NVMe (2019)
* NVMe to PCIe adaptor

I successfully installed the following OSs on the – non bootable - NVMe SSD, and I am able to boot any one of them through Clover :
* Windows 10
* Windows 11
* Linux Mint LMDE 4
* Kali Linux
* Ubuntu Server [Edit - 21/06/2022]
* Emmabuntüs [Edit - 01/09/2022]

I created the Clover USB key using Boot Disk Utility v2.1.2018 rev.023b and Clover v.4961.

For reference, from the Clover key, it takes 13 s (thirteen seconds) to start Windows 11 and 15 s for Kali Linux. Not bad for a 13 year old rig !

Clover is a wonderful tool and I can only encourage others who have a legacy platform to use it !

Attachment (Clover v.4961 & BDU v2.1.2018 rev.023b)
[CLOVER.zip](https://github.com/CloverHackyColor/CloverBootloader/files/9752638/CLOVER.zip)

**Comment by evans-max** [comment]
Hi,
My Dell T7610 Windows 10 pro 64 bit does not detect the Nvme. I wrote the image on USB from the link given.

https://www.hamishmb.com/files/Downloads/misc/Clover.img.7z.

Pls help me with how should I proceed further.

**Reply by KevinMoorman** [reply]
If you get [BDU version v2.1.2018 rev.023b](https://web.archive.org/web/20180615122122/http://cvad-mac.narod.ru/index/bootdiskutility_exe/0-5) and use it to obtain Clover v4961 and format your USB flash drive, then you still need to activate the NVME driver for Clover.
For me the USB device is drive G: and I went to this folder:
G:\EFI\CLOVER\drivers-Off\drivers64UEFI
where I copied the driver "NvmExpressDxe-64.efi"
Then, I pasted that driver into this folder:
G:\EFI\CLOVER\drivers64

So presumably anybody following this would need to do the same steps, but instead of drive G: you would use the drive letter that has your newly-created copy of Clover v4961 on it.
Try that & give us feedback how it works, I think this is the extent of it.

**Comment by evans-max** [comment]
how do i get nvme proper driver from ?
I have  pm981a Nvme Samsung1tb

**Reply by KevinMoorman** [reply]
For me the USB device is drive G: and I went to this folder:
G:\EFI\CLOVER\drivers-Off\drivers64UEFI
where I copied the driver "NvmExpressDxe-64.efi"
Then, I pasted that driver into this folder:
G:\EFI\CLOVER\drivers64

**Comment by Morbius01** [comment]
@[evans-max](https://github.com/evans-max)

Perhaps you should first explain what you wish to do as precisely as possible.

How have you installed your NVMe SSD in your Dell system ?  Directly on the motherboard ?  Or through an NVMe PCIe adapter ?

**Comment by Morbius01** [comment]
Reporting success in installing another OS with Clover on my non-bootable NVMe SSD drive in my “legacy” system : **Emmabuntüs** ! :smiley:

I have now installed the following OSs and all are bootable with Clover :
● Windows 10
● Windows 11
● Linux Mint LMDE
● Kali Linux
● Ubuntu Server
● Emmabuntüs

**Comment by KevinMoorman** [comment]
Hey I was able to find that [BDU version v2.1.2018 rev.023b](https://web.archive.org/web/20180615122122/http://cvad-mac.narod.ru/index/bootdiskutility_exe/0-5).
Sure enough, was able to open the Options/Configuration menu and press "Check Now" and then beside DL to select "4961" version of Clover.
Pressed OK, then plugged in my 16GB Lexar USB flash drive, selected it, and then pressed "Format Disk".
Thanks much for the advice! I will try right away on my HP xw8600 server, which only has "legacy boot" (BIOS -- non UEFI).

**Reply by Morbius01** [reply]
Clever from you to find BDU v2.1.2018 rev.023b from the Wayback machine !

Once you have created a Clover USB key that works, you will be half way from success !  The second step consists in installing your OS on your non-bootable NVMe SSD drive from Clover.

Please report any success, or difficulty you may have if you need any further help.

**Reply by GTHInfo** [reply]
I successfully created the Clover and booting starts, but NVME is displaying as Misc Device and not recognized at the start up, Can you please help resolve this, screes shot attache of that clover Boot Manager, Thanks,
![20231025_194723 1](https://github.com/CloverHackyColor/CloverBootloader/assets/148346154/f3e978d4-77d2-4e1d-aa5d-f0493074dd3e)

**Comment by Temovich567** [comment]
я скачал вашу прошивку, но после заставки биоса . выходит только надпись приветствую в кловере , и идет сканирование входов. на этом все останавливается.
буду пробовать другой кловер

**Comment by 10Veronika12** [comment]
Попробуй в ютубе найти Ra1nUSB и перекинь Файлы Кловер на свой.после
установки даже с драйверами мучиться не будешь

Сб, 4 марта 2023 г. в 18:25, Temovich567 ***@***.***>:

> я скачал вашу прошивку, но после заставки биоса . выходит только надпись
> приветствую в кловере , и идет сканирование входов. на этом все
> останавливается.
> буду пробовать другой кловер
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CloverHackyColor/CloverBootloader/discussions/491#discussioncomment-5202559>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANCULQLPEVTVD3FKKOQQBZ3W2NNIBANCNFSM5LQPANYA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <CloverHackyColor/CloverBootloader/repo-discussions/491/comments/5202559@
> github.com>
>
-- 
null

**Reply by Temovich567** [reply]
Спасибо за помощь, но уже решил свою проблему без кловера. 
Описание проблемы ( может кому поможет) 
Решил установить nvme самсунг Рм981а через переходник в  pcex4  на материнскую плату х79 из Китая  c Xeon. Слота м2 нет. 
Материнка ССД увидела, даже разрешила сделать его первым в загрузке. 
Склонировал с сата ССД виндоус 7 в MBR - он не грузится загрузчик не видит путь. 
Пытаюсь установить виндоус 11 пишет этот диск не может быть загрузочным. 
Пытаюсь сделать через Кловер. Один вариант кловера зависает, другой загружается но не по одному из вариантов загрузки не может ничего загрузить так и мигает курсором. 
( откуда он берёт список загрузки и как его править, информации не нашёл) 
В итоге закинул в биос драйвер нвме (4 вариант не сжатый в конец списка дхе драйверов все как в ЮТУБ) 
Теперь в биосе нвме самсунг отображается как PATA SSD
Думаю основная проблема была в том что диск был в МБР. Отформатировал его в GPT. С помощью программы Wintohdd сделал флешку и установил виндоус в системе gpt  разметки диска.
После установки Виндоус 11 на NVME ,в биос кроме PATA SSD появляется " загрузка с "Windows Uefi NVM samsung " дословно не помню. выбирать надо именно это первым, если выбрать PATA SSD загрузка не идет.
 С помощью этой программы AOMEI Partition Assistant 9.13.1, создал на NVME новую неразмеченную область.
зашел в старый виндоус 7 на SSD. и от туда сделал клонирование на NVME. эта программа сразу клонирует и меняет MBR на GPT.
Путь загрузки правда не работает, надо из под виндоус 11 на NVME редактировать путь загрузки с помощью EasyBCD 2.4
Получается две системы на одном NVME, в биос появляется две записи для загрузки "Windows Uefi NVM samsung ".
загружается выбор системы , если выбираешь Windows11 грузится дальше если запись Windows 7 уходит в перезагрузку и только потом грузит без выбора Windows 7.
Пробросить пути загрузки С NVME на обычные диски не получается, как и в обратную сторону .
только если в биос менять загрузку: с обычных дисков: тогда можно с них снова загрузится ( правда винда запускает проверку дисков , что то ей не нравится)

**Reply by Temovich567** [reply]
Thanks for the help, but already solved my problem without clover.
Description of the problem (can help someone)
I decided to install nvme samsung Rm981a through an adapter in pcex4 on x79 motherboard from China with Xeon. There is no m2 slot.
The motherboard of the SSD saw, even allowed to make it the first in the boot.
I cloned windows 7 from sata ssd to MBR - it does not boot, the bootloader does not see the path.
I'm trying to install Windows 11, it says this disk cannot be bootable.
I'm trying to do through Clover. One version of the clover freezes, the other one loads, but according to one of the boot options it cannot load anything and blinks the cursor.
(where does he get the download list from and how to edit it, I did not find any information)
As a result, I threw the nvme driver into the BIOS (the 4th option is not compressed to the end of the list of dhe drivers, everything is like in YouTube)
Now in bios nvme samsung is displayed as PATA SSD
I think the main problem was that the disk was in the ICBM. Formatted it to GPT. Using the Wintohdd program, I made a USB flash drive and installed Windows in the gpt disk partitioning system.
After installing Windows 11 on NVME, in addition to the PATA SSD, "booting from" Windows Uefi NVM samsung" appears in the BIOS.
  With this program, AOMEI Partition Assistant 9.13.1 created a new unpartitioned area on NVME.
I went into the old Windows 7 on the SSD. and from there made cloning on NVME. this program immediately clones and changes MBR to GPT.
The boot path doesn't really work, you need to edit the boot path from under Windows 11 on NVME using EasyBCD 2.4
It turns out two systems on one NVME, two entries appear in the BIOS for loading "Windows Uefi NVM samsung".
the system selection is loaded if you choose Windows11 it loads further if the Windows 7 entry goes into reboot and only then loads without selecting Windows 7.
Forwarding boot paths With NVME on regular disks does not work, as well as in the opposite direction.
only if you change the boot in the BIOS: from ordinary disks: then you can boot from them again (although Windows starts checking disks, it doesn’t like something)

**Comment by Morbius01** [comment]
@Temovich567
Please keep in mind that this is mainly an English language forum and thread.
Which means that, if you have a problem, you are strongly advised to describe it in English in order to maximize your chances of receiving help from others.

**Comment by siliconbeaver** [comment]
I did try Clover boot on old PCs.

It worked fine on HP EliteDesk 800 G1 (2013 PC, DDR3). I tried Windows 7. no issue. PCI adapter with NVMe Samsung PM961 512GB.
But it wouldn't work on ASUS M2A-VM (2007 PC, DDR2). CPU AMD, PCI adapter with NVMe Samsung PM961 512GB (was thinking to save Windows 7 install). Clover boot was hanging over, a cursor was blinking on black/blank screen. failed to reach the boot menu.

Any suggestion? Thanks,

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
tried again.

With downloading your CLOVER.zip. tested exactly v2.1.2018 rev.023b and Clover v.4961. but no luck. only the cursor blinking. it's hanging there forever.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
my 3rd try on two XP machine still failed. It's a test after I have read over ten times your instruction posted on winraid. every word has been dissected. (made CLOVER the XP machine, copied NvmExpressDxe-64.efi… )

any BDU and CLOVER versions worked fine on my HP EliteDesk 800 G1/Win7. My concern that, graphics/video of XP machine could be an issue. NOT sure. I had tried previous versions as early as Clover-1174-X64/CloverISO-1022.

The two XP motherboard are
ASUS M2A-VM (Mar 2007. DDR2)
https://www.asus.com/us/supportonly/m2a-vm/
ECS 7100PVT (Oct 2007. DDR2)
https://www.ecs.com.tw/en/Product/Motherboard/GF7100PVT-M3/specification

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Thank you for the *.zip file.

Finally I made it.
That's surprising. I almost gave up right before that time as my previous tests all failed.

CloverISO-4411 to 5070 worked on my ASUS M2A-VM motherboard, AMD CPU; ECS 7100PVT motherboard, Intel CPU.
my present version is CloverISO-5070 (latest version), on a 2GB USB drive with format size 1GB. (BDU default setting is 200MB. That can be misleading)

Windows 7/10/11. either is working, boot via Clover to the OS. my disk is GPT one. It's ok to run an NVMe SSD with pre-installed OS.
Windows 7 seems better for old hardware. compact and quick. Windows 11 can save install of old PC drivers. but start time still long. Of course, much faster than IDE HDD or SATA SSD.

Again, thank you for this post.

**Reply by GTHInfo** [reply]
New BDU uvrsion {BDU_v2.1.2022.030b.X64} and Clover CloverV2-5155 are available now, but when created a boot disk, it can be edited because files are not shown in windows 11 explorer and booting screen also not showing boot options, just black at the start up, tried clover 4961 also, it didn't work, can somebody help to create newest versions with New BDU and Clover to work with Windows 11 in NVMe drives please, Thanks,

**Comment by 10Veronika12** [comment]
здравствуйте извините что долго не отвечал,редко пользуюсь почтой странно
что не загрузилось.я просто писал любой клевер,потом вскрывать флешку и
писал туда все из другой флешки но там стоял уже rainusb.есть варианты и с
интел и амд.на следующей неделе доберусь до компа и смогу создать полную
флешку,если не получится у вас.

ср, 18 окт. 2023 г., 15:32 GTHInfo ***@***.***>:

**Reply by GTHInfo** [reply]
Please share with us, if you do it successfully, still I was ubable to create one with newer BDU and Clover versions to successfully boot from a NVME drive with an adaptor card for a old PC. Thanks,

**Reply by GTHInfo** [reply]
The problem is NVME is displayed as Misc Device and not shown in the boot options, please see the attachment, Thanks,
![Uploading 20231025_194723 1.jpg…]()

**Reply by Morbius01** [reply]
See my answer below.

**Comment by Morbius01** [comment]
@ [GTHInfo](https://github.com/GTHInfo)
I was away from the forum for a while. I just came back.

Have you succeeded in installing your OS and booting it from Clover ?

If not, since you posted several messages in this thread, I am willing to help you.
But could you specify :
- your platform (motherboard, CPU, the PCIe version and size (number of lanes))
- the OS you are trying to install. If it is Windows 11, could you indicate which version ?  A link to the ISO file you used to create your USB Installer would help.

**Reply by nineascii** [reply]
**Thank-you for this tutorial!**
I have a Samsung 1Tb Evo NVMe SSD plugged into a generic PCIe adapter card.
I connected it into an old Dell Optiplex 9010 w/ 3rd gen Intel i7 CPU.
I downloaded the "Pre-prepared Clover with NVMExpressDxe.efi preinstalled for NVME support" from:
https://www.hamishmb.com/miscellaneous-downloads/
extracted the file, then I used rufus-4.3p to write it to an old USB stick. I did not need to make any changes to the image.
Booted the Dell Optiplex 9010 in a UEFI mode. **It works!**  (so happy) I booted to a windows 10 that I had cloned over from a sata ssd onto the NVMe drive. The only thing plugged into the machine is the NVMe via a PCIe adapter, and the USB stick.
3500 MB/s, not too shabby. 
_P.S. - when the bios was trying to boot via Legacy instead of UEFI the cursor just flashed and did nothing. UEFI was needed._

**Reply by Morbius01** [reply]
@ [nineascii](https://github.com/nineascii)
While I am happy that you succeeded with the Clover key you created using Hamish's image, I need to say that I have tried the Clover image from Hamish earlier and that it has never worked for me (system frozen with blinking cursor). Possibly because my system is not UEFI compatible.
You mentioned that your system is UEFI compatible, and that it only worked in UEFI mode, not in legacy mode.
In his blog, Hamish specified that the system he used to test Clover was a UEFI system, too.

My advice to others would be that, if you try to make a Clover key from this image, and that it does not work for you, don't waste too much time on it. Especially if your system is  not UEFI compatible.

**Reply by GTHInfo** [reply]
Thanks for the above reply, I created the clover as you explained earlier and it didn't work as I explained earlier, NVMe is displayed as Misc Device and that boot option does not show in the start.
My motherboard is Samsung G41 and Core 2 Duo processor, PCI Express Version is 1.0 and 16 size adaptor, NVMe is Kimtigo 128 GB, please help resolve this,

**Reply by Morbius01** [reply]
_"...NVMe is displayed as Misc Device and that boot option does not show in the start...."_
I don't understand.
Could you post an image of the screen you get after booting on your newly created Clover key ?

Reading your earlier posts, it looks like you wish to install Windows 11 on your NVMe SSD. Is this correct ?
If it is, could you post a link to the page with the ISO file you have downloaded to create you Windows 11 Installer USB drive ?

**Reply by GTHInfo** [reply]
Yes I want to boot my Windows 11 from NVMe drive which is already cloned from my normal HDD. I made the clover as you explained and it worked, but although I copied the NVMe Drivers, this option is not active in the clover menu, as I explained NVMe is displayed in the HDD list as MISC device and it is not actively shown as a boot drive, I attach how it shows in the list of boot drives in clover menu.
I don't have the link I downloaded the windows ISO file, but it is working fine and fully up to date.
![20231025_194723 1](https://github.com/CloverHackyColor/CloverBootloader/assets/148346154/5cc36363-4f45-47b4-9ecc-ad886f15f294)

**Reply by Morbius01** [reply]
What you are trying to do, boot with Clover from a Windows 11 which you have cloned from a HDD to your NVMe SSD, is something that I have never succeeded in doing !

All I can explain you is how to re-install Windows 11 on your NVMe SSD so that it will be bootable with Clover.

**Reply by GTHInfo** [reply]
OK tell me please, my problem is, 
01. My G41 DDR3 Motherboard doesn't have NVMe slot and bought a PCIe adaptor for that,
2. Although NVMe is fixed in the adaptor which is not shown in the BIOS.
3. Windows 11 was clonned which worked in normal M2 SSD but not with this NVMe one.
4. Now I want to boot my machine with Windows 11 from this NVMe SSD fixed in the adaptor.
5. I thought Clover would show this NVMe option in Clover Boot, but it didn't.
Please tell me any option for getting this done, Thanks,

**Reply by Morbius01** [reply]
“3. Windows 11 was clonned which worked in normal M2 SSD but not with this NVMe one. “
I must say I didn’t understand this. Earlier, you said that you cloned Windows 11 from your HDD to your NVMe SSD. Let’s move on…

IMPORTANT : As you know, at the beginning of the Windows installation process, the installer asks you (1) if you want to re-build Windows, or (2) if you want to start a fresh installation of Windows.
You may like to try (1) since you have cloned Win 11 from your HDD to your SSD, but it might not work. And, in this case, you will have to start again doing (2). So, my advice is that you start a fresh installation (2), and then transfer your user files from your HDD to your new Windows 11 on your SSD. This is what I always did.

I strongly suggest that, before you start, you disconnect any other storage device in your PC, such as any HDD.
You should only have :
• Your NVMe SSD (installed through the adaptor plugged in the PCIe slot)
• Your Clover USB drive
• Your Windows 11 installation USB drive

You did not indicate how you created your Clover key. Did you use the files I posted in the attachment at the beginning of this discussion ?
If you did, you should have a Clover version 4961 key.
IMPORTANT : There is one more thing you need to do. In your Clover key :
Copy the ‘EFI/CLOVER/drivers-Off/drivers64UFI/NvmExpressDxe-64.efi’ driver to:
• 'EFI/CLOVER/drivers64’
and to:
• ‘EFI/CLOVER/drivers64UEFI’
Unless you have done it before, of course ! This is essential.

Now, start your PC with the Clover USB drive inserted only (not the Windows installation USB drive). This Clover USB drive should remain plugged into during the entire installation process of Windows.
Let your system boot Clover.
On the main Clover Boot Manager menu, navigate to the last entry ‘Exit Clover’.
Then, insert your Windows installation USB drive into a USB port.
Then select:
Exit Clover → Boot Maintenance Manager (and not Boot Manager as you did before) → Boot from file → Windows 11 USB drive → \<efi> → \<boot> → bootx64.efi
This should start the Windows installation process.
After the first reboot, you can remove the Windows installation USB drive and boot again on the Clover key.
This time, you should see a new icon named ‘Boot Microsoft EFI from EFI’ or similar. Select this icon to continue the Windows installation process. And do the same for all subsequent reboots.

At the end, you will get Windows 11 installed on your NVMe SSD. And you boot on it from the Windows icon on the main Clover Boot Manager menu.

**Reply by GTHInfo** [reply]
Thanks for the step by step explanations, your way was successful and I tested couple of days, that's why got a little late to make a reply. What I previously said was, cloned Windows on a M2 SSD was shown in the clover icons and was bootable, similarly I was unable to do it for NVMe SSD. Your way worked, Thanks so lot for that.
Is there anyway to transfer all my apps and settings at once to this new SSD, without setting up one bu one configuration, that's easier.
Also I used your clover key Clover v.4961. for this as you explained.
Hopefully I may need to post again for certain things going forward.
Thanks once again for this great help.

**Comment by noabody** [comment]
Have you tried with **BDU_v2.1.2022.030b.X64.zip** and **Clover-5132-X64.iso.7z**?  Looks like driver is in `EFI\CLOVER\drivers\off\NvmExpressDxe.efi` and needs to be copied to `EFI\CLOVER\drivers\BIOS` and `EFI\CLOVER\drivers\EFI`.

I have very old Biostar TA790GX3 A2+ and AMD Phenom X4 9950 BE without USB3 and EFI support.  Clover-5132 let me boot from PCI USB3 card and I bet it will work with NVMe on PCIEx4 (don't have one to test).

Except I don't get a menu with any version of Clover ISO image after 5132.  Blinking underline cursor at upper left of screen at 800x600, and mode switches to maybe 1024x768 but only the blinking cursor, no menu.

Let me run Windows 11 to Go created with RUFUS, though.  Pretty funny.

![impossible](https://github.com/CloverHackyColor/CloverBootloader/assets/29848078/5b7b92ae-3987-4539-8b8a-648903b1f979)

**Comment by improbir** [comment]
> After several weeks of research, I finally managed to successfully install Clover on a 2009 (!) legacy BIOS (non UEFI) system.
> 
> My system :
> 
> * MB : Gigabyte MA790XT-UD4P (2009)
> * CPU : AMD Phenom II X4 955 (2009)
> * SSD : TRANSCEND TS1MTE220S 1TB NVMe (2019)
> * NVMe to PCIe adaptor
> 
> I successfully installed the following OSs on the – non bootable - NVMe SSD, and I am able to boot any one of them through Clover :
> 
> * Windows 10
> * Windows 11
> * Linux Mint LMDE 4
> * Kali Linux
> * Ubuntu Server [Edit - 21/06/2022]
> * Emmabuntüs [Edit - 01/09/2022]
> 
> I created the Clover USB key using Boot Disk Utility v2.1.2018 rev.023b and Clover v.4961.
> 
> For reference, from the Clover key, it takes 13 s (thirteen seconds) to start Windows 11 and 15 s for Kali Linux. Not bad for a 13 year old rig !
> 
> Clover is a wonderful tool and I can only encourage others who have a legacy platform to use it !
> 
> Attachment (Clover v.4961 & BDU v2.1.2018 rev.023b) [CLOVER.zip](https://github.com/CloverHackyColor/CloverBootloader/files/9752638/CLOVER.zip)

bro with your zip files i was able to boot into clover bootloader manager.. but the problem is without PCI nvme I'm not able to boot into neither windows nor Ubuntu via my sata ssd through clover bootloader gui. is it possible to boot from sata ssd?

**Comment by Mark-Greenville** [comment]
Hi!  I'm a newbie here.  Trying to create a USB boot disk in Windows 10 using the same versions of BDU and Clover that you used.  Can set option in BDU to download latest Clover and it finds online version OK, but cannot get the local ISO to work.  Tried putting ISO in to same folder as BDU and also sub-folder, and setting local repository to that folder, but does not find image there, so cannot create boot disk.  I tried the Clover file as ISO, as ZIP, and as TAR, but nothing works for me.  How did you get BDU to find that local Clover distribution file???  Will appreciate your help!  Thanks.

**Comment by ATOM3699** [comment]
Hi, 
I have almost the same motherboard, around a 2009 vintage GA-MA790GP-UD4H  (GP instead of XT), but I can't get anywhere trying to get the motherboard to boot in to a NVMe or a SSD install in the system.  I've tried several different version of BDU and Clover, from v4166 to the newsest 5161.

All I get is a text screen that identifies the motherboard and processor and it says some things in blue that might be the addresses of the boot on the SSD and NVMe disks.  The only option is to select a EFI HARD DRIVE.  And I noticed that I need to format the USB stick as exFAT,
to even get the text screen.

Are the some things you need to setup in the config.plist file?  Do I need a seperate MBR bootloader to load up Clover?  What I am I missing here?

**Comment by n-a-b-s-t-e-r** [comment]
> After several weeks of research, I finally managed to successfully install Clover on a 2009 (!) legacy BIOS (non UEFI) system.
> 
> My system :
> 
>     * MB : Gigabyte MA790XT-UD4P (2009)
> 
>     * CPU : AMD Phenom II X4 955 (2009)
> 
>     * SSD : TRANSCEND TS1MTE220S 1TB NVMe (2019)
> 
>     * NVMe to PCIe adaptor
> 
> 
> I successfully installed the following OSs on the – non bootable - NVMe SSD, and I am able to boot any one of them through Clover :
> 
>     * Windows 10
> 
>     * Windows 11
> 
>     * Linux Mint LMDE 4
> 
>     * Kali Linux
> 
>     * Ubuntu Server [Edit - 21/06/2022]
> 
>     * Emmabuntüs [Edit - 01/09/2022]
> 
> 
> I created the Clover USB key using Boot Disk Utility v2.1.2018 rev.023b and Clover v.4961.
> 
> For reference, from the Clover key, it takes 13 s (thirteen seconds) to start Windows 11 and 15 s for Kali Linux. Not bad for a 13 year old rig !
> 
> Clover is a wonderful tool and I can only encourage others who have a legacy platform to use it !
> 
> Attachment (Clover v.4961 & BDU v2.1.2018 rev.023b) [CLOVER.zip](https://github.com/CloverHackyColor/CloverBootloader/files/9752638/CLOVER.zip)

I downloaded Morbius' CLOVER.ZIP and uploaded it to VirusTotal.  It red-flagged the following: Backdoor.Win32.Androm.XTA@4z809t,  Trojan.Win32.Agent.dg and Malware.Heuristic.1008
